app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentAssuranceConnector.scala

```diff
@@ -34,7 +34,7 @@ class AgentAssuranceConnector @Inject()(@Named("agent-assurance-baseUrl") baseUr
   override val kenshooRegistry: MetricRegistry = metrics.defaultRegistry
 
   def hasAcceptableNumberOfClients(regime: String)(implicit hc: HeaderCarrier): Future[Boolean] = {
-    monitor(s"ConsumedAPI-AgentAssurance-hasAcceptableNumberOfClients-GET") {
+    monitor(s"ConsumedAPI-Agent-Assurance-hasAcceptableNumberOfClients-GET") {
```

Comment:
we should not change names of existing metrics, it breaks history and current dashboards

Reply:
OK, Ill change it back